### PR TITLE
Alternative description

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #12665.

same text in two fields on one form (new menu item - list all news feed categories -categories tab)

Thanks to @MartijnMaandag for reporting this issue

### Summary of Changes
Go to New Menu - Menu Item Type = List All News Feed Categories
Go to tab "Categories" and you will find two fields with the name Top Level Category Description. 
I think it is better to rename the second one into "alternative description"
And the tooltips have been updated to match

### CODE REVIEW

File: administrator/language/en-GB/en-GB.ini [hunk 1]
```diff
@@ -322,7 +322,7 @@ JGLOBAL_DOWN="Down"
 JGLOBAL_EDIT_ITEM="Edit item"
 JGLOBAL_EDIT_PREFERENCES="Edit Preferences"
 JGLOBAL_EMAIL="Email"
-JGLOBAL_EMPTY_CATEGORIES_DESC="Show or hide categories that contain no articles and no subcategories."
```
Comment (zero-24) [inline administrator/language/en-GB/en-GB.ini]: Should this be removed?

File: administrator/language/en-GB/en-GB.ini [hunk 2]
```diff
@@ -322,7 +322,7 @@ JGLOBAL_DOWN="Down"
...
+JGLOBAL_FIELD_CATEGORIES_DESC_DESC="If you enter some text in this field, it will replace the Top Level Category Description, if it has one."
```
Comment (zero-24) [inline administrator/language/en-GB/en-GB.ini]: to be sure Is DESC_DESC expected?

File: administrator/language/en-GB/en-GB.ini (hunk 1 above)
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.ini]: copy paste error

File: administrator/language/en-GB/en-GB.ini (hunk 2 above)
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.ini]: copy past error

**Comment by jeckodevelopment** [comment]
I have tested this item :white_check_mark: successfully on 81a38bbe20f07285f5561494a9a14aaa66ff9e86<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12677">issues.joomla.org/tracker/joomla-cms/12677</a>.</sub>

**Comment by conconnl** [comment]
I have tested this item :white_check_mark: successfully on 684369c37bba6358a498d7818153d2444c8bb95c<br /><br />Test successful at PBF NL<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12677">issues.joomla.org/tracker/joomla-cms/12677</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12677">issues.joomla.org/tracker/joomla-cms/12677</a>.</sub>

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**